spring-cloud-kubernetes-config/src/test/java/org/springframework/cloud/kubernetes/config/ConfigMapsSpringBootTest.java

```diff
@@ -75,11 +76,13 @@ public void beforeTest() {
 			.once();
 	}
 
+	@Ignore
```

Comment:
The ignored test cases, are actually pretty valid for the core module, which is responsible for injecting the client. I would move them there (unignored ofc).